views/type-of-appeal.njk

```diff
@@ -0,0 +1,64 @@
+{% set pageTitleName = i18n.pages.appealType.header %}
+{% extends "layout.njk" %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% import "helpers/text-helpers.njk" as text %}
+{% import "helpers/button-helpers.njk" as button %}
+
+{% block content %}
+
+    {% if error %}
+        {{ govukErrorSummary({
+            titleText: i18n.errorSummary.titleText,
+            errorList: [{
+                text: error,
+                href: "#appealType"
+            }]
+        }) }}
+    {% endif %}
+
+    {% set options = [] %}
+    {% for item in i18n.pages.appealType.options %}
+        {% set examples =
+            govukDetails({
+            summaryText: i18n.pages.appealType.exampleSummaryText,
+            html: text.bold(  item.example.name ) + " " + text.parentheses(item.example.description)
+        }) %}
+
+        {% set options = (options.push(
+            {
+                value: item.id,
+                html: text.bold( item.name ) + " " + text.parentheses(item.description),
+                hint: { html: examples }
+            }), options) %}
```

Comment:
I would avoid adding any logic on the views or rely on the en.json file for constructing the appeal options...I think we should have a file declaring all the options available with hints and any other thing we may need and just pass that to the view